src/helpers/json_to_predreq.py

```diff
@@ -1,12 +1,19 @@
-from jaqpotpy.datasets import JaqpotpyDataset
 import pandas as pd
+from src.helpers.recreate_featurizer import recreate_featurizer
+from jaqpotpy.datasets import JaqpotpyDataset
 
 
-def decode(request, model):
+def decode(request):
     df = pd.DataFrame(request.dataset['input'])
     independent_features = request.model['independentFeatures']
     smiles_cols = [feature['key'] for feature in independent_features if feature['featureType'] == 'SMILES']
     x_cols = [feature['key'] for feature in independent_features if feature['featureType'] != 'SMILES']
+
+    if request.model['extraConfig']['featurizers']:
```

Comment:
maybe the extra config here is undefined? Can you test with a model that has no extra config?

Reply:
Added a line in case no featurizers are sent with the model. Then featurizer = None.
The extraConfig will allways be sent from the models even if it's empty